napari/utils/_testsupport.py

```diff
@@ -34,6 +34,7 @@ def napari_plugin_manager(monkeypatch):
     from unittest.mock import patch
 
     import napari
+    import napari.plugins.io
```

Comment:
is this needed?

Reply:
yes. For running part of test to allow use monkeypatch few lines bellow. (If you run full test set then `napari.plugins.io` is already loaded, but there are single tests which running need this explicit import).

Reply:
yep needed